src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_BinaryOperators.vb

```diff
@@ -126,9 +140,34 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return left
         End Function
 
-        Private Function RewriteBinaryOperatorSimple(node As BoundBinaryOperator) As BoundNode
+        Private Shared Function GetLeftOperand(binary As BoundBinaryOperator, ByRef optimizeForConditionalBranch As Boolean) As BoundExpression
+            If optimizeForConditionalBranch AndAlso (binary.OperatorKind And BinaryOperatorKind.OpMask) <> BinaryOperatorKind.OrElse Then
```

Comment:
><> BinaryOperatorKind.OrElse [](start = 104, length = 28)

nit: I didn't understand why we check for "not OrElse" rather than just checking for "AndAlso" (seems more direct)